cmd/controller/app/server.go

```diff
@@ -48,9 +48,10 @@ func Run(s *ServerRunOptions) error {
 	// Controller Runtime Controllers
 	ctrl.SetLogger(klogr.New())
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                  scheme,
-		MetricsBindAddress:      s.MetricsAddr,
-		Port:                    9443,
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: s.MetricsAddr,
+		},
```

Comment:
makes sense to me but has this changed in between 1.28.4 and 1.28.8?

Reply:
This change is brought from the new controller-runtime version: https://github.com/kubernetes-sigs/scheduler-plugins/pull/711#issuecomment-2051100858

Reply:
thanks, then LGTM